src/JSInterop/Microsoft.JSInterop/src/DotNetDispatcher.cs

```diff
@@ -93,6 +93,8 @@ public static void BeginInvoke(string callId, string assemblyName, string method
             catch (Exception ex)
             {
                 syncException = ExceptionDispatchInfo.Capture(ex);
+                syncException.SourceException.Data.Add(JSRuntimeBase.AssemblyNameKey, assemblyName);
+                syncException.SourceException.Data.Add(JSRuntimeBase.MethodKey, methodIdentifier);
```

Comment:
This mechanism seems kind of elaborate. What's the reason for not adding `assemblyName` and `methodIdentifier` as two extra params to `EndInvokeDotNet`?

Reply:
Yeah, lets do that, I thought end invoke dotnet was part of the public contract or called across boundaries, I realize now it's not, so I'll just do that.